disperser/controller/dispatcher.go

```diff
@@ -554,9 +554,10 @@ func (d *Dispatcher) NewBatch(ctx context.Context, referenceBlockNumber uint64)
 
 	copy(batchHeader.BatchRoot[:], tree.Root())
 
-	batchHeaderHash, err := batchHeader.Hash()
```

Comment:
What was the reason for moving this after the timer ends?

Reply:
It's more close to batch header handling, as the stuff above is building merkle tree and measuring it's latency.